src/olympia/api/tests/test_authentication.py

```diff
@@ -221,3 +221,8 @@ def test_verify(self):
         assert response.status_code == 200
         data = json.loads(response.content)
         assert data['token'] == token
+
+    def test_invalid_user_id(self):
+        token = self.client.generate_api_token(self.user, user_id=-1)
+        with self.assertRaises(AuthenticationFailed):
+            self._authenticate(token)
```

Comment:
Unfortunately the logic in DRF-JWT checked `is_active` which we don't set and didn't check `deleted` since that isn't standard so deleted user's tokens still worked (oops).